teachopencadd/talktorials/T015_protein_ligand_docking/talktorial.ipynb

```diff
@@ -4,231 +4,206 @@
    "cell_type": "markdown",
```

Comment:
<ul><li>Using a combination of three free software <strong>projects</strong></li><li>a comparably easy task<strong>:</strong> the reproduction of</li><li><strong>Considering </strong>certain sidechains <strong>as flexible </strong></li><li> in <strong>this</strong> situation</li></ul><p> </p><p>Add a link to PDBFixer here in case anybody is interested in knowing how to overcome these potential pitfalls.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>